taskcluster/tc-build-utils.sh

```diff
@@ -312,7 +370,13 @@ do_nuget_build()
 
   cd ${DS_DSDIR}/native_client/dotnet
 
-  cp ${DS_TFDIR}/bazel-bin/native_client/libdeepspeech.so nupkg/build
+  # We copy the generated SO files into the Nuget runtime dirs
+
+  mkdir -p nupkg/runtimes/win-x64/native/  
```

Comment:
once things are green: it might be good to avoid hardcoding arches like that and maybe iterate over some list